bin/os_pp.sh

```diff
@@ -3,52 +3,53 @@
 
 # Installing needed tools
 # NB: this bash script run in travis where sudo is required. Thus we must put sudo before the usual pip command
-sudo pip install python-openstackclient==3.11.0
+sudo -H pip install cryptography=="2.2.2"
+sudo -H pip install python-openstackclient=="3.17.0"
 
 # Building the OS Instance
 cd ./bin || exit
 
 # Environmental Variables for building with Terraform apply
+export TF_VAR_auth_url="$OS_AUTH_URL"
 export TF_VAR_username="$OS_USERNAME"
 export TF_VAR_password="$OS_PASSWORD"
-export TF_VAR_auth_url="$OS_AUTH_URL"
-export TF_VAR_user_domain_id="$OS_USER_DOMAIN_ID"
-export TF_VAR_domain_id="$OS_DOMAIN_ID"
+export TF_VAR_domain_name="$OS_DOMAIN_NAME"
+export TF_VAR_tenant_name="$OS_TENANT_NAME"
 export TF_VAR_region_name="$OS_REGION_NAME"
-export TF_VAR_project_id="$OS_PROJECT_ID"
-export TF_VAR_project_name="$OS_PROJECT_NAME"
 export TF_VAR_api_version="$OS_IDENTITY_API_VERSION"
 export TF_VAR_os_pool_name="$OS_POOL_NAME"
-export TF_VAR_ssh_key_pub="${HOME}/.ssh/id_rsa.pub"
+export TF_VAR_ssh_key_pub="../keys/id_rsa.pub"
+export TF_VAR_ssh_key_prv="../keys/id_rsa"
 export TF_VAR_current_version=$CURRENT_VERSION
 export TF_VAR_kubenow_image_name=$IMAGE_NAME
 
 # Parsing KubeNow image ID of newly crated one. \s option will look for exact match
-image_id=$(openstack image list | grep "\skubenow-$CURRENT_VERSION\s" | awk '{print $2}')
-export TF_VAR_kubenow_image_id=$image_id
+kn_image_id=$(openstack image list | grep "\skubenow-$CURRENT_VERSION\s" | awk '{print $2}')
+echo -e "Latest Kubenow image ID: $kn_image_id"
+export TF_VAR_kubenow_image_id=$kn_image_id
 
 # Parsing some needed values for spawning Openstack instance with terraform
 os_image_id=$(openstack image list | grep "Ubuntu 16.04 Xenial Xerus" | awk '{print $2}')
+echo -e "Ubuntu 16.04 Xenial Xerus image ID: $os_image_id"
 export TF_VAR_os_image_id=$os_image_id
 
+# Parsing default network id in our Openstack provider
 network_id=$(openstack network list | grep -i "default" | awk '{print $2}')
+echo -e "Default Network ID: $network_id"
 export TF_VAR_network_id=$network_id
```

Comment:
Refactoring all terraform variables given the new rc file from our Openstack provider.